### PR TITLE
use requestedFormat in agherant

### PR DESCRIPTION
**Pull request by ael-code**
- requestedFormat() function moved to weban/webserver_utils.py
  - updated webant and agherant search function to make use of requestedFormat()

related to #28

### CODE REVIEW

**Comment by boyska** [comment]
this should be rebased

**Comment by ael-code** [comment]
> this should be rebased
done.

File: webant/webant.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
+<<<<<<< HEAD
```
Comment (boyska) [inline webant/webant.py]: mh, rebase went bad?

Comment (ael-code) [inline webant/webant.py]: > mh, rebase went bad?

I'm very ashamed for this error. Sorry
anyway it was fixed in 055f58f

**Comment by boyska** [comment]
webserver_utils? why webserver_utils? I can't see any webserver-ish thing going on there. wasn't that `utils.py` before?

**Comment by ael-code** [comment]
> wasn't that utils.py before?

Yes, you're right.

I have changed my mind. 
Actually, I do not deeply understood the difference between these two files.
But the function seems to match the webserver_utils.py desc:
>This module provides some function to make running a webserver a little easier

**Comment by boyska** [comment]
It's about the distinction between an http server and an application ;)
requestedFormat is on the level of a Flask application, not on the one of a http server

**Comment by ael-code** [comment]
moved one more time.
Now should be fine.

**Comment by boyska** [comment]
ok. I am now noticing that you are using `application/opensearchdescription+xml` for everythin opensearch-related.
This is wrong: it should only be used for `description.xml` files.

Sorry for pointing one thing at a time and not all together

**Comment by ael-code** [comment]
> ok. I am now noticing that you are using application/opensearchdescription+xml for everythin opensearch-related.

I have pointed this out in https://github.com/insomnia-lab/libreant/issues/31#issuecomment-70081802

anyway we could leave ```opensearch``` for human friendly selection, but what about the standard mime-type? which one is the correct one for opensearch result?

**Comment by boyska** [comment]
I don't see where you pointed it. valid mime-types for the opensearch format on `/search` can be `text/xml` or `application/rss+xml`. Or of course "fancy" ones, like `opensearch`. But `application/opensearchdescription+xml` cannot be.

**Comment by ael-code** [comment]
fixed the mimetype issue

**Comment by boyska** [comment]
perfect, merging

**Pull request merged**